from0k2bp.tex

```diff
@@ -305,10 +308,16 @@
 
 But, crucially for the rest of this document, this new commitment scheme
 \emph{does} have a homomorphism:
-\[C(r_1, a_1) + C(r_2, a_2) = r_1H + a_1G + r_2H + a_2G = (r_1 + r_2)H + (a_1 + a_2)G = C(r_1+r_2, a_1+a_2)\]
-In words: ``the sum of the commitments to and is equal to a commitment
-to , as long as you choose the randomness for each of the first two
-commitments so that their sum is equal to randomness for the third.''
+
+\begin{align*}
+C(r_1, a_1) + C(r_2, a_2) & = r_1H + a_1G + r_2H + a_2G  \\
+                          & = (r_1 + r_2)H + (a_1 + a_2)G \\
+                          & = C(r_1+r_2, a_1+a_2)
+\end{align*}
+
+In words: ``the sum of the commitments to \[a_1\] and \[a_2\] is equal to a commitment
```

Comment:
This and many other corrections of this type are greatly appreciated.

Reply:
Oh but: checking the pdf, the use of `[` seems to have resulted in `a_1` and `a_2` being printed on separate lines, so this specific format didn't work. Why not use `$`?

Reply:
Yes. `$` signs work better. Fixed.